src/cli.coffee

```diff
@@ -35,6 +35,7 @@ watchCmd = program
   .option('-p, --port [port]', 'if a `server` option was specified, define on which port
  the server would run')
   .option('-d, --debug', 'print verbose debug output to stdout')
+  .option('-l, --stdin', 'listen to stdin and shut down when stdin closes')
```

Comment:
I guess `commander` thinks the `-` option is just a leftover argument, so it gets treated differently and does *not* act the same as passing `--stdin`. Perhaps using the `-l` short option would work best. Unless you can poach the `-` off and make it act like `--stdin`.